<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookEditView</class>
 <widget class="QWidget" name="BookEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书编辑</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f5f7fa;
    font-family: &quot;Microsoft YaHei&quot;, &quot;Arial&quot;;
    font-size: 14px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>124</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelIsbn">
       <property name="text">
        <string>ISBN：*</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditIsbn">
       <property name="styleSheet">
        <string notr="true">QLineEdit, QComboBox {
    padding: 8px;
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    background-color: white;
    min-height: 20px;
}

QLineEdit:focus, QComboBox:focus {
    border-color: #409eff;
    outline: none;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 6px solid #666;
    width: 12px;
    height: 12px;
}</string>
       </property>
       <property name="placeholderText">
        <string>请输入ISBN号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTitle">
       <property name="text">
        <string>书名：*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditTitle">
       <property name="styleSheet">
        <string notr="true">QLineEdit, QComboBox {
    padding: 8px;
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    background-color: white;
    min-height: 20px;
}

QLineEdit:focus, QComboBox:focus {
    border-color: #409eff;
    outline: none;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 6px solid #666;
    width: 12px;
    height: 12px;
}</string>
       </property>
       <property name="placeholderText">
        <string>请输入图书名称</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAuthor">
       <property name="text">
        <string>作者：*</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditAuthor">
       <property name="styleSheet">
        <string notr="true">QLineEdit, QComboBox {
    padding: 8px;
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    background-color: white;
    min-height: 20px;
}

QLineEdit:focus, QComboBox:focus {
    border-color: #409eff;
    outline: none;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 6px solid #666;
    width: 12px;
    height: 12px;
}QLineEdit, QComboBox {
    padding: 8px;
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    background-color: white;
    min-height: 20px;
}

QLineEdit:focus, QComboBox:focus {
    border-color: #409eff;
    outline: none;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 6px solid #666;
    width: 12px;
    height: 12px;
}</string>
       </property>
       <property name="placeholderText">
        <string>请输入作者姓名</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPublisher">
       <property name="text">
        <string>出版社：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditPublisher">
       <property name="styleSheet">
        <string notr="true">QLineEdit, QComboBox {
    padding: 8px;
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    background-color: white;
    min-height: 20px;
}

QLineEdit:focus, QComboBox:focus {
    border-color: #409eff;
    outline: none;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 6px solid #666;
    width: 12px;
    height: 12px;
}</string>
       </property>
       <property name="placeholderText">
        <string>请输入出版社名称</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelPublishYear">
       <property name="text">
        <string>出版年份：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditYear">
       <property name="placeholderText">
        <string>请输入出版年份</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelCategory">
       <property name="text">
        <string>分类：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QComboBox" name="comboBoxCategory">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>古典文学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>外国文学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>现代文学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>计算机科学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>自然科学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>社会科学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>其他</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelTotalCopies">
       <property name="text">
        <string>总数量：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QSpinBox" name="spinBoxTotalCopies">
       <property name="suffix">
        <string>本</string>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelAvailableCopies">
       <property name="text">
        <string>可借数量：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QSpinBox" name="spinBoxAvailableCopies">
       <property name="suffix">
        <string>本</string>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelLocation">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-weight: ;
    color: #333;
    padding: 5px 0px;
}

/* 必填字段标签样式 */
QLabel[text*=&quot;*&quot;] {
    color: #f56c6c;
}</string>
       </property>
       <property name="text">
        <string>位置：</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditLocation">
       <property name="placeholderText">
        <string>例如：A103</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="2">
      <widget class="QPushButton" name="pushButtonOk">
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButtonOk {
    background-color: #409eff;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 8px 20px;
    font-weight: 500;
}

QPushButton#pushButtonOk:hover {
    background-color: #66b1ff;
}

QPushButton#pushButtonOk:pressed {
    background-color: #3a8ee6;
}</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>124</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
